kafka-win-config.yml.sample

```diff
@@ -0,0 +1,45 @@
+integration_name: com.newrelic.kafka
```

Comment:
Should we create a copy for windows or use universal linux one?

Reply:
I think it should be the same in *nix and windows. I think there's another attribute to distinguish between win and *nix